README.txt

```diff
@@ -33,97 +34,118 @@ dictionary that stores the top K terms for each document (this is used for Rocch
 
 We begin by loading the previously stored data: the PostingList file cursor values, all document lengths for subsequent normalisation, and all doc_ids with their 
 top K most common terms, which will be used for optimisation. We will then parse the query file, with the first line being the actual query and subsequent lines 
-(which represent documents marked as relevant by the law experts) into an array relevant_docids. We will now begin search.
+(which represent documents marked as relevant by the law experts) into an array called relevant_docids. We will now begin search.
 
 Firstly, the parse_query function takes in the query and calls split_query to obtain words to process into terms later on. This process also determines the search 
 type. Here, in split_query, we are splitting the original query given from the query file into either words of length 1, or phrases (identified by double 
 inverted commas in a phrasal query). In the process, if we encounter the Boolean Retrival keyword "AND", we know this is a boolean query and set is_boolean_query 
-to True. Otherwise, we will process this term in a free-text query. Please note that phrasal queries will be performed as part of processing in boolean query.
-However, if the query is purely phrasal, we will treat it as a free-text query, which will then be processed as a phrasal query. A phrasal query can easily be 
-identified by spaces (" ") in its term, because split_query produces the respective phrases whenever it encounters the double inverted commas. At the end of
-this function, we will have a sequence of words and phrases. Once done, the free-text or boolean query (which contains any phrasal queries) is then executed.
+to True. Otherwise, we will process this term in a free-text query. 
+
+Please note that phrasal queries (appearing with "AND") will be performed as part of a boolean query. However, if the query is purely phrasal, we will process it 
+as a free-text query, which will then be processed as a purely phrasal query. In other words, if the entire query contains only one phrase and nothing else, it 
```

Comment:
As long as there's a phrase, its processed as part of a boolean query

Reply:
Yo can you help update the parts I changed in README and BONUS, especially for boolean-related stuff?